src/scheduled_tasks.rs

```diff
@@ -94,7 +94,7 @@ pub async fn setup(context: LemmyContext) -> Result<(), LemmyError> {
       delete_old_denied_users(&mut context.pool()).await;
       update_instance_software(&mut context.pool(), context.client())
         .await
-        .inspect_err(|e| warn!("Failed to update instance software: {e}"))
+        .map_err(|e| warn!("Failed to update instance software: {e}"))
```

Comment:
Had to get rid of all these `.inspect`, since they're actually only nightly / unstable.

Reply:
Nope it was stabilized in 1.76

https://doc.rust-lang.org/std/result/enum.Result.html#method.inspect_err

Reply:
That can't be right, or I'm missing something else: https://woodpecker.join-lemmy.org/repos/129/pipeline/6302/4#L2860

That's on rust:1.77

Reply:
Thats strange because Im developing with stable rust and never saw that error since 1.76. I also tried just now to build a release docker image locally, and it worked fine on 1.76.

Reply:
Same failure again, weird. Gonna merge like this then.